pkg/dashboard/functiontemplates/generator/generator.go

```diff
@@ -256,14 +256,14 @@ func (g *Generator) buildFunctionTemplates(functionDirs []string) ([]*functionte
 			"runtime", configuration.Spec.Runtime)
 		functionTemplates = append(functionTemplates, functionTemplate)
 
-		// HACK: allow python 3.7, 3.8, 3.9, 3.10, 3.11 share the same functions to avoid specific examples per runtime version
+		// HACK: allow different python versions to share the same functions to avoid specific examples per runtime version
 		runtimeName, runtimeVersion := common.GetRuntimeNameAndVersion(configuration.Spec.Runtime)
 		if runtimeName == "python" &&
 			runtimeVersion == "3.9" &&
 			functionName == "helloworld" {
 
 			// add helloworld function example to python 3.7+
-			for _, runtimeCopy := range []string{"python:3.7", "python:3.8", "python:3.9", "python:3.10", "python:3.11"} {
+			for _, runtimeCopy := range []string{"python:3.7", "python:3.8", "python:3.9", "python:3.10", "python:3.11", "python:3.12"} {
```

Comment:
would be enough, no?
```suggestion
			for _, runtimeCopy := range []string{"python:3.9", "python:3.10", "python:3.11", "python:3.12"} {
```

Reply:
@liranbg this is 1.13.x where we still support them